app/middleware/createGpViewModel.js

```diff
@@ -29,6 +30,7 @@ function createGpViewModel(req, res, next) {
       choicesId: gpData.choicesId,
       location: gpData.location,
       facilities: parseFacilities(gpData.facilities),
+      services: servicesMapper(gpData.services),
```

Comment:
We've got a variety of function names for carrying out similar operations of changing the shape of the data:

parseOpeningTimes
parseFacilities
servicesMapper
getBookOnlineLink

it would be good to get these similarly named, whether they are all mappers, parsers or getters I don't mind.